src/Psa/Output.purs

```diff
@@ -8,16 +8,16 @@ module Psa.Output
   ) where
 
 import Prelude
-import Data.Foldable (foldl, any)
-import Data.Maybe (Maybe(..), fromMaybe)
-import Data.Tuple (Tuple(..), fst)
+import Data.Array as Array
 import Data.Set as Set
-import Data.String as Str
 import Data.StrMap as StrMap
-import Data.Array as Array
+import Data.String as Str
 import Node.Path as Path
-import Psa.Types (PsaOptions, PsaError, PsaAnnotedError, PsaPath(..), PsaResult, Position, Filename, Lines,
-                  compareByLocation)
+import Data.Foldable (foldl, any)
+import Data.Maybe (Maybe(..), fromMaybe)
+import Data.String (Pattern(Pattern))
+import Data.Tuple (Tuple(..), fst)
+import Psa.Types (PsaOptions, PsaError, PsaAnnotedError, PsaPath(..), PsaResult, Position, Filename, Lines, compareByLocation)
```

Comment:
The idea is that the qualified and implicit imports are the one you edit by hand so I put them first.